modules/security-container-content-external-scanning.adoc

```diff
@@ -236,7 +236,7 @@ annotation. Be sure to replace the values for `<token>`, `<openshift_server>`,
 $ curl -X PATCH \
   -H "Authorization: Bearer <token>" \
   -H "Content-Type: application/merge-patch+json" \
-  https://<openshift_server>:8443/oapi/v1/images/<image_id> \
+  https://<openshift_server>:8443/apis/image.openshift.io/v1/images/<image_id> \
```

Comment:
Should we use 6443 port, e.g. `https://<openshift_server>:6443`

Reply:
Yes, 6443 is used for kube-apiserver, 8443 is for openshift-apiserver